tests/test_cnn/test_swish.py

```diff
@@ -1,5 +1,5 @@
 import torch
-from torch.nn.functional import sigmoid
+from torch import sigmoid
```

Comment:
import torch.nn.functional as F

F.sigmoid(...)

Reply:
Is it any difference between torch.sigmoid and F.sigmoid?

Reply:
They are the same.

Reply:
Done